app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -113,9 +130,8 @@ export default function DocumentDownloadMenu(props) {
         ) {
```

Comment:
@frozenhelium `url.includes('?retry=true')` in the previous line won't match url like `?journal=true&retry=true`.